tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -77,3 +77,27 @@ Wait Until Build Status Is
     [Arguments]    ${namespace}    ${build_name}    ${expected_status}=Complete    ${timeout}=20 min
     Wait Until Keyword Succeeds    ${timeout}    1 min
     ...    Build Status Should Be    ${namespace}    ${build_name}    ${expected_status}
+
+Provoke Image Build Failure
+    [Documentation]    Starts New Build after some time it fail the build and return name of failed build
+    [Arguments]    ${namespace}    ${build_name_includes}    ${build_config_name}    ${container_to_kill}
+    ${build} =    Search Last Build    namespace=${namespace}    build_name_includes=${build_name_includes}
+    Delete Build    namespace=${namespace}    build_name=${build}
+    ${failed_build_name} =    Start New Build    namespace=${namespace}
+    ...    buildconfig=${build_config_name}
+    ${pod_name} =    Find First Pod By Name    namespace=${namespace}    pod_start_with=${failed_build_name}
+    Wait Until Build Status Is    namespace=${namespace}    build_name=${failed_build_name}
+    ...    expected_status=Running
+    Wait Until Container Exist  namespace=${namespace}  pod_name=${pod_name}  container_to_check=${container_to_kill}
+    Run Command In Container    namespace=${namespace}    pod_name=${pod_name}    command=/bin/kill 1    container_name=${container_to_kill}
+    Wait Until Build Status Is    namespace=${namespace}    build_name=${failed_build_name}
+    ...    expected_status=Failed    timeout=5 min
+    [Return]    ${failed_build_name}
+
+Delete Failed Build And Start New One
+    [Documentation]    It will delete failed build and start new build
+    [Arguments]    ${namespace}    ${failed_build_name}    ${build_config_name}
+    Delete Build    namespace=${namespace}    build_name=${failed_build_name}
+    ${build_name} =    Start New Build    namespace=${namespace}
+    ...    buildconfig=${build_config_name}
+    Wait Until Build Status Is    namespace=${namespace}    build_name=${build_name}
```

Comment:
for clarity, could you write the `${expected_status}` argument?

Reply:
Updated.